sdk/kubernetesconfiguration/Microsoft.Azure.Management.KubernetesConfiguration/src/Microsoft.Azure.Management.KubernetesConfiguration.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.reference.props" />
+  <!-- Please do not move/edit code below this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.KubernetesConfiguration</PackageId>
+    <Description>Provides Microsoft Azure KubernetesConfiguration management operations including the ability to create, update and delete runbooks and schedules.</Description>
+    <Version>3.8.2-preview</Version>
```

Comment:
Suggest `1.0.0-preview.1` for a new module. See https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md

Reply:
Fixed.